source/elements/oneMKL/source/domains/data_fitting/terms.rst

```diff
@@ -0,0 +1,324 @@
+.. _terms:
+
+Common Terms
+============
+
+.. contents::
+    :local:
+    :depth: 2
+
+Glossary
+--------
+
+Assume we need to interpolate a function ``f(x)`` on the ``[a, b)`` interval using splines.
+
+Let's break ``[a, b)`` into sub-intervals by ``n`` points :math:`x_i`
+called partition points or simply **partition**.
+**Function values** at these points (:math:`y_i = f(x_i), i=1,\cdots,n`) are also given.
+
+Spline has ``k`` degree if it can be expressed by the following polynomial:
+
+.. math::
+  P\left( x \right) =
+  c_{1} +
+  c_{2}\left( x - x_i \right) +
+  c_{3}{\left( x - x_i \right)}^2 + \cdots +
+  c_{k-1}{\left( x - x_i \right)}^k.
+
+Splines are constructed on :math:`[x_i, x_{i+1}), i=1,\cdots,n-1` sub-intervals.
+So, for each sub-interval there are following polynomials:
+
+.. math::
+  P_i\left( x \right) =
+  c_{1,i} +
+  c_{2,i}\left( x - x_i \right) +
+  c_{3,i}{\left( x - x_i \right)}^2 + \cdots +
+  c_{k-1,i}{\left( x - x_i \right)}^k.
+
+:math:`c_{j,i}, j=1,\cdots,k, i=1,\cdots,n` are called spline **coefficients**.
+
+Function is interpolated at points of [a, b).
+Such points are called interpolation sites or simply **sites**. Sites might or migtn't equals to partition points.
+
+Mathematical Notation in the Data Fitting Component
+---------------------------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Concept
+     - Mathematical Notation
+   * - Partition
+     - :math:`\left\{ x_i \right\}_{i=1,\cdots,n}`,
+       where :math:`a = x1 < x2< \cdots < xn = b`..
+   * - Uniform partition
+     - Partition :math:`\left\{ x_i \right\}_{i=1,\cdots,n}`
+       which meets the following condition:
+
+       :math:`x_{i+1} - x_i = x_i - x_{i-1}, i=2,\cdots,n-1`
+   * - Quasi-uniform partition
+     - Partition :math:`\left\{ x_i \right\}_{i=1,\cdots,n}`
+       which meets the constraint with a constant ``C`` defined as:
+       :math:`1 \le M / m \le C`,
+
+       where
+
+       :math:`M = max_{i=1,\cdots,n-1} (\Delta_i)`,
```

Comment:
I like to encase words like max.min etc that show up in math mode in \text{} so that it looks a little better:  IT is your preference, but something to consider:

:math:`M = \text{max}_{i=1,\cdots,n-1} (\Delta_i)`,

Reply:
I should have used \dots here as well instead of \cdots